sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/implementation/SnapshotImpl.java

```diff
@@ -140,7 +140,12 @@ public Mono<Void> awaitCopyStartCompletionAsync() {
                 }
                 return result;
             })
-            .repeatWhen(longFlux -> Flux.interval(manager().serviceClient().getDefaultPollInterval()))
+            .repeatWhen(longFlux ->
+                longFlux
+                    .flatMap(
+                        index -> Mono.delay(
+                            ResourceManagerUtils.InternalRuntimeContext.getDelayDuration(
+                                manager().serviceClient().getDefaultPollInterval()))))
```

Comment:
`Flux.interval(ResourceManagerUtils.InternalRuntimeContext.getDelayDuration(manager().serviceClient().getDefaultPollInterval()))`
The default interval 1ms in playback mode is too small for Flux.interval to work.

Reply:
nit, will `.delaySubscription(...).repeat()` work? https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollingUtil.java#L82

Reply:
It works, thanks